composer.json

```diff
@@ -48,5 +48,8 @@
         "psr-4": {
             "Buzz\\Test\\": "tests"
         }
+    },
+    "conflict": {
+        "symfony/polyfill-php80": "*"
```

Comment:
Why is this needed?

Reply:
From @chris-doehring : 
due to the combination of different symfony packages, I had to set the package symfony/polyfill-php80 as conflict so it will be ignored for installation (credits: laravel/framework#33045 (comment))

Reply:
I still dont understand why it should be ignored for installation.

Reply:
I will try to reproduce the error I got while running one of the CI pipelines.

Reply:
I don't have the time to do it now, but will check it out this evening

Reply:
I think the problem is that we run `composer install` on php7.4 
but the tests run on php8-dev from `tommymuehle/docker-alpine-php-nightly`

https://github.com/kriswallsmith/Buzz/blob/7b43248c5bad1eb8db805cd1fb8ed77ee891ffc7/.github/workflows/ci.yml#L102-L125

Reply:
Good catch! I assumed it was still based on php 7.4, as it's docker hub readme suggested it: https://hub.docker.com/r/tommymuehle/docker-alpine-php-nightly/